index.js

```diff
@@ -41,9 +41,9 @@ export default function() {
         i       = directions.length,
         coords,
         scrollTop  = document.documentElement.scrollTop ||
```

Comment:
this default assignment should be deleted because you get the document.body from the function anyway and if the given rootElement has scrollTop 0, it gets evaluated incorrectly
(same below) 

btw amazing timing! @tbeckenhauer submitted the same fix (#234) just a couple of hours after you!